auto_migrate.py

```diff
@@ -0,0 +1,22 @@
+import sqlite3
+
+
+def auto_migrate():
+    with (sqlite3.connect('culture-popper.db')) as conn:
```

Comment:
this is nice because you don't have to do conn.disconnect as it auto disconnects when using with